apps/researcher/src/lib/community.test.ts

```diff
@@ -92,3 +97,115 @@ describe('isAdmin', () => {
     expect(isAdmin(memberships)).toEqual(false);
   });
 });
+
+describe('sortCommunities', () => {
+  const communities = [
+    {
+      id: 'community2',
+      name: 'Community 2',
+      slug: 'community-2',
+      imageUrl: 'https://example.com/image.png',
+      createdAt: 1690000000000,
+    },
+    {
+      id: 'community1',
+      name: 'Community 1',
+      slug: 'community-1',
+      imageUrl: 'https://example.com/image.png',
+      createdAt: 1600000000000,
+    },
+    {
+      id: 'community4',
+      name: 'Community 4',
+      slug: 'community-4',
+      imageUrl: 'https://example.com/image.png',
+      createdAt: 1680000000000,
+    },
+    {
+      id: 'community3',
+      name: 'Community 3',
+      slug: 'community-3',
+      imageUrl: 'https://example.com/image.png',
+      createdAt: 1650000000000,
+    },
+    {
+      id: 'community5',
+      name: 'Community 5',
+      slug: 'community-5',
+      imageUrl: 'https://example.com/image.png',
+      createdAt: 1670000000000,
+    },
+  ];
+  it('sorts communities by name ascending', () => {
+    const sortedCommunities = sortCommunities(
+      communities,
+      CommunitySortBy.NameAsc
+    );
+    const sortedCommunitiesIds = sortedCommunities.map(
+      community => community.id
+    );
+    expect(sortedCommunitiesIds).toEqual([
```

Comment:
Curious: does `toEqual()` test the order of the elements in the array? Or should we use `toStrictEqual()` instead?

Reply:
It does test the order, but I will change it to `toStrictEqual()`